python/foglamp/plugins/south/modbustcp/modbustcp.py

```diff
@@ -175,9 +175,13 @@ def plugin_poll(handle):
                 source_port = int(handle['port']['value'])
             except:
                 raise ValueError

```

Comment:
For proper diagnosis, an error message must be associated with ValueError.

Reply:
Added logging and now ValueError is being raised with error message.